jakarta.batch.arquillian.exec.example/test.properties

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Before we had test.properties in .gitignore and were copying from jakarta.batch.arquillian.exec/tck.default.sleep.time.properties

Now we're just committing the file though we still have some .gitignore references.   I guess we could duplicate the file though it'd be better to copy at execution time. 

Either way itt'd be nice to fix this before merging.

Reply:
I committed the test.properties file into this project because it's only an example. It shouldn't be used as a maven artifact, people should only use it as a template for their TCK execution project. In that case I believe it makes more sense if people just commit their test.properties file directly rather than generate it at execution time from a template.

The best solution would be if the maven project could download the template from some maven artifact and then copy it to test.properties. But there's no such maven artifact. All projects that contain a file that could be used as a template are plain POM artifacts, which don't contain it in the deployed maven artifact. 

Maybe we should create a template test.properties file in the `com.ibm.jbatch.tck` artifact so that it can be copied from it? It could also be used in the `com.ibm.jbatch.tck.exec` RI execution project, which now contains its own copy.

If you agree, I'll add the template test.properties file there and modify both RI and Arquillian exec projects to copy it from the maven dependency.

Reply:
OK. So I added the `tck.default.sleep.time.properties` template file into the `com.ibm.jbatch.tck` artifact and adapted  arquillian exec parent to copy that file from the `com.ibm.jbatch.tck` artifact to test.properties if that file doesn't exist yet. The file `test.properties` remains ignored by git so the old semantics remains - the test.properties file is generated based on a template and can be changed later for local execution, without committing it to git. 

Projects based on the Arquillian parent created by those who want to execute the TCK against the implementation of their choice can add the generated `test.properties` file to git as their project will be in a different repository.